### ISSUE TITLE
Turned ranking training

### ISSUE THREAD

**Issue by junsuhas** [pengxingang/PocketXMol#3]
Hi! Thanks for sharing your code.
I think it's the best of the diffusion models.

Before I answer your question, let me share a test I did.
I've done some testing with Posebuster and it seems that reducing the noise to a certain level doesn't make much difference, but it does reduce the time.
In my tests, I used noise at **100**, **50**, and **10**, and pose at 50. I used **poseboff** as the data was provided.

The result.

![Image](https://github.com/user-attachments/assets/22f0ee49-5b29-4385-af95-bfe0ff6714a3)

Thank you.

As we proceeded with the docking, we ran into the possibility issue of whether or not the result was a good binding affinity and we could select that compound.
So I was wondering about the tuned ranking training code.
Do you have any plans to share it?

**Comment by pengxingang** [comment]
Tuning the ranker is straightforward—it involves a simple fine-tuning of PocketXMol using only the confidence loss on the generated decoy poses. We plan to share the training code for the ranker in the future.

Additionally, we did observe some correlations between the ranker scores and various indicators of protein–ligand pair quality, such as Vina scores, binding activity, and pocket correctness.

Notably, the self-confidence score also serves as a valuable metric for assessing the reliability of protein–ligand pairs.
